cfgov/jinja2/v1/_includes/molecules/quote.html

```diff
@@ -8,21 +8,26 @@
 
    Creates markup for quote when given. Typically used in insets.:
 
-   settings: (required) Mandatory settings for this inset
+   value.quote:      String with quote for the text.
 
-   settings.quote:      String with quote for the text.
-
-   settings.citation:   String with the author or citation of text.
+   value.citation:   String with the author or citation of text.
 
    ========================================================================== #}
 
-{% macro render(settings) %}
+{% if not value %}
+    {% do global_dict.update({
+      'quote': {
+        'quote': 'This is a quote.',
```

Comment:
Can we rename this `body` so the final object is `quote.body` and it matches the markup?